.travis.yml

```diff
@@ -1,40 +1,52 @@
+dist: xenial
 language: python
-sudo: required
-group: deprecated-2017Q4
+sudo: false
 env:
-  - PYTHON_VERSION=2.7
+  - PYTHON_VERSION=3.6
 before_install:
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
```

Comment:
Any reason not to use the latest?

Reply:
Nop, will change.

Reply:
Well, looks like libcfortran is different between version; returned so tests pass